src/routes/categories/containers/CategoriesContainer.js

```diff
@@ -67,9 +68,17 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
     return { status: 'loading' }
   }
 
+  const refreshProps = {
+    cityCode: route.city,
+    language: route.language,
+    path: route.path,
+    navigation: ownProps.navigation
+  }
+
   if (route.status === 'languageNotAvailable') {
     if (languages.status !== 'ready') {
```

Comment:
Can't you check for `languages.status === 'error'` and then use the `languages.message` as error message and code?

Reply:
No because the languages are also not available if they are loading :man_shrugging:

Reply:
You have excluded the case above if they are loading.

Reply:
Right... :man_facepalming: